Dockerfile-debian.template

```diff
@@ -32,6 +32,8 @@ RUN set -ex; \
         libpng-dev \
         libpq-dev \
         libxml2-dev \
+        libmagickwand-dev \
+        libmagickcore-dev \
```

Comment:
`libmagickwand-dev` already depends on `libmagickcore-dev`, so this is not needed.

Reply:
No, Alpine does not split the package. This does only affect Debian.